migrations/2020-11-04-070016_sp_laporan/up.sql

```diff
@@ -0,0 +1,33 @@
+-- Counting Laporan for Numbering
+CREATE OR REPLACE FUNCTION sp_laporan_count_nomor(_id_jenis int, _id_satker int, _tanggal_laporan timestamp)
```

Comment:
```suggestion
CREATE OR REPLACE FUNCTION sp_laporan_count_month(_id_jenis int, _id_satker int, _tanggal_laporan timestamp)
```

Reply:
ok kang

di query sebelumnya seperti ini

Input (contoh)
==> 2020/11/03

Query
`tanggal_laporan >= date_trunc('month', _tanggal_laporan) AND tanggal_laporan < date_trunc('month', _tanggal_laporan + interval '1' month)`

Output
`tanggal_laporan >= 2020/11/01 00:00:00 AND tanggal_laporan < 2020/12/01 00:00:00`

Atas saran dari Kang Adit
kira2 querynya akan seperti ini

`WHERE EXTRACT(MONTH FROM tanggal_laporan) == 11 AND EXTRACT(YEAR FROM tanggal_laporan) == 2020`

parameter sp akan berubah